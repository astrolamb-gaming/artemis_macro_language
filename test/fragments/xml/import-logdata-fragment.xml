<mission_data version="1.0">
    <mission_description>  Multiplayer mission with Jump drive ships. Capture the most eggs by towing them to your station. Eggs can only be towed at impulse. Comms Button at start of mission must be pressed to join game. </mission_description>
    <imports>
        <import name="import-logdata-xml.txt" />
        <import name="import-logdata-json.txt" as="json" />
    </imports>
    <start>
        <expand>
        <set_variable name="fromjson" value="${count_json}"/>
        <set_variable name="fromxml" value="${count_xml}"/>
        </expand>
    </start>
</mission_data> 