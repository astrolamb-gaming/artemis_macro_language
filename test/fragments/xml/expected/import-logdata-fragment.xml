<mission_data version="1.0">
  <mission_description>  Multiplayer mission with Jump drive ships. Capture the most eggs by towing them to your station. Eggs can only be towed at impulse. Comms Button at start of mission must be pressed to join game. </mission_description>
  <start>
    <set_variable name="fromjson" value="5"/>
    <set_variable name="fromxml" value="67"/>
  </start>
</mission_data>